sql_generators/glean_usage/templates/events_stream_v1.query.py

```diff
@@ -77,12 +78,19 @@ def main(submission_date, billing_project, destination_table, temp_dataset, slic
                 ],
             ),
         )
-        print(
+        logging.info(
             f"Writing sample id {min_sample_id} to {max_sample_id} to {temp_table}: {job.path}"
         )
         job.result()
         temp_tables.append(job.destination)
 
+    # update destination table schema if the source has changed
+    temp_table_schema = client.get_table(temp_tables[0]).schema
+    if not temp_table_schema != destination_table.schema:
```

Comment:
This condition seems like it would only pass if the schemas are the same (double negative = positive)?